### PR TITLE
Updated References in all files

### PR DESCRIPTION
**Pull request by bitwisebro**
updated class import references. corrected requirement installed command in readme, added instruction to create folder to avoid file not found error

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -124,6 +124,7 @@ venv.bak/
 .mypy_cache/
 .dmypy.json
 dmypy.json
+myMain.py
```
Comment (MrFlamez) [inline .gitignore]: pls keep your own individual gitignore local. i have my own gitignore version.

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+charset-normalizer==2.0.12
+idna==3.3
 Pillow==9.0.1
-praw==7.5.0
+praw==7.6.0
+prawcore==2.3.0
 requests==2.27.1
+update-checker==0.18.0
+urllib3==1.26.9
+websocket-client==1.3.3
```
Comment (MrFlamez) [inline requirements.txt:10]: are these moduls really neccessary?

File: src/spreddit.py [hunk 3]
```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 
-from instagram import Instagram, ProcessingError, InstagramLoginError
-from reddit import Reddit            
+from .instagram import Instagram, ProcessingError, InstagramLoginError
```
Comment (MrFlamez) [inline src/spreddit.py]: from .instagram import Instagram
from .errors import ProcessingError, InstagramLoginError

File: .gitignore (hunk 1 above)
Comment (bitwisebro) [reply .gitignore]: Yeah sure thanks

File: requirements.txt (hunk 2 above)
Comment (bitwisebro) [reply requirements.txt:10]: Yes. I used pip freeze to generate this requirement file

File: src/spreddit.py (hunk 3 above)
Comment (bitwisebro) [reply src/spreddit.py]: I'd update that

**Comment by MrFlamez** [comment]
Please update your code. I fixed all of the problems. After that, i think we can close this pull request

**Pull request merged**